introspection-engine/introspection-engine-tests/tests/re_introspection/mod.rs

```diff
@@ -1319,13 +1319,13 @@ async fn multiple_many_to_many_on_same_model(api: &TestApi) -> TestResult {
     let final_dm = expect![[r#"
         model B {
           id        Int @id @default(autoincrement())
-          custom_A  A[]
+          custom_A  A[] @relation("AToB")
           special_A A[] @relation("AToB2")
```

Comment:
We don't consider the first one in a group of ambiguous relations unambiguous anymore, we're stricter (as per commit message).